package.json

```diff
@@ -60,19 +70,20 @@
     "karma-webpack": "^1.7.0",
     "load-grunt-tasks": "^3.5.2",
     "minimist": "^1.2.0",
+    "rimraf": "^2.6.2",
     "sinon": "^1.17.4",
-    "webpack": "^1.13.1",
-    "webpack-dev-server": "^1.14.1",
     "url-search-params": "^0.6.1",
-    "typescript": "^2.0.3"
+    "webpack": "^3.10.0",
+    "webpack-dev-server": "^1.16.5"
   },
   "browser": {
     "./lib/adapters/http.js": "./lib/adapters/xhr.js"
   },
   "typings": "./index.d.ts",
   "dependencies": {
     "follow-redirects": "^1.2.5",
-    "is-buffer": "^1.1.5"
+    "is-buffer": "^1.1.5",
```

Comment:
I don't see this used here, or was it there before ?

Reply:
It's just the comma, as `pkginfo` was added 😂

Reply:
In the utils for now, later on we might get rid of it when moving to lodash